src/test/java/am/rockstars/controller/ProductControllerTest.java

```diff
@@ -27,25 +33,55 @@
     @Test
     void findProductById() throws Exception {
         //Test data
+        final List<FeatureInfo> features = new ArrayList<>();
+        features.add(new FeatureInfo("feature1"));
+        features.add(new FeatureInfo("feature2"));
+
+        final List<TagInfo> tags = new ArrayList<>();
+        tags.add(new TagInfo("tag1"));
+        tags.add(new TagInfo("tag2"));
+
         final ProductPayload productPayload = ProductPayload.builder()
```

Comment:
I think we can use randomObject it will help to create ProductPayload and keep clean ex. https://github.com/f985/aaa/blob/941735fdf8b805bc483f887936219e94bd18694f/src/test/java/am/rockstars/controller/HeaderControllerTest.java#L90